src/widgets/range-input/__tests__/range-input-test.js

```diff
@@ -0,0 +1,335 @@
+import ReactDOM from 'react-dom';
+import AlgoliasearchHelper from 'algoliasearch-helper';
+import rangeInput from '../range-input.js';
+
+jest.mock('react-dom', () => ({
+  render: jest.fn(),
+}));
+
+describe('rangeInput', () => {
+  const attributeName = 'aNumAttr';
+  const createContainer = () => document.createElement('div');
+  const instantSearchInstance = { templatesConfig: undefined };
+  const createHelper = () =>
+    new AlgoliasearchHelper(
+      {
+        search() {},
+        addAlgoliaAgent() {
+          return {};
+        },
+      },
+      'indexName',
+      { disjunctiveFacets: [attributeName] }
+    );
+
+  afterEach(() => {
+    ReactDOM.render.mockReset();
```

Comment:
Couldn't we rely on snapshots here? I'm not sure that relying on mocks helps much or does it?

Reply:
I mock the render method in order to be able to use the snapshot on it.

Reply:
Ok!